recipes/tirmite/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Please update it to `$PYTHON setup.py install --single-version-externally-managed --record=record.txt`.

Reply:
Fixed in latest commit

Reply:
Not sure why linux build test is failing

Reply:
I fixed the linux build. The issue was that libgcc was in the build reqs section instead of run reqs. Just need to wait for the osx travis build to finish.